### PR TITLE
Dac logic

### PR DESCRIPTION
**Pull request by rusefillc**

### CODE REVIEW

File: firmware/auxout.cpp [hunk 1]
```diff
@@ -55,7 +121,16 @@ void AuxOutThread(void*)
...
         {
-            SetAuxDac(ch, GetLambda(ch));
+            float lambda = GetLambda(ch);
+            // todo: make translation configurable
+            if (lambda < 0.7)
+                lambda = 0.7;
+            if (lambda > 1.3)
+                lambda = 1.3;
+
+            // "oil ration" calibration
```
Comment (mck1117) [inline firmware/auxout.cpp:131]: what is "oil ration"

Comment (rusefillc) [reply firmware/auxout.cpp:131]: "Oil ration" is a reference to the $14 gauge with massive typos in documentation

Comment (mck1117) [reply firmware/auxout.cpp:131]: can we avoid reprinting known massive mistranslations?

Comment (mck1117) [reply firmware/auxout.cpp:131]: maybe a link to the forum page instead

Comment (rusefillc) [reply firmware/auxout.cpp]: pushed

File: firmware/auxout.cpp [hunk 2]
```diff
@@ -55,7 +121,16 @@ void AuxOutThread(void*)
...
+            float lambda = GetLambda(ch);
+            // todo: make translation configurable
+            if (lambda < 0.7)
+                lambda = 0.7;
+            if (lambda > 1.3)
+                lambda = 1.3;
+
+            // https://rusefi.com/forum/viewtopic.php?f=4&t=2410 for now
+            float voltage = 1 - (1.3 - lambda) / 0.6;
+            SetAuxDac(ch, voltage);
```
Comment (mck1117) [inline firmware/auxout.cpp:133]: perhaps this is a feature for the future, but do we want to do something different if the sensor isn't alive (yet)?

Comment (rusefillc) [reply firmware/auxout.cpp:133]: That's above my paygrade. Any chance we can merge as is and you could implement anything on top?

**Pull request merged**